plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationWizard.java

```diff
@@ -148,6 +168,25 @@ public void run() {
 		return success;
 	}
 
+	protected Map<IProject, Collection<IBuildConfig>> getBuildConfigs(Collection<IResource> resources) {
+		Map<IProject, Collection<IBuildConfig>> projects = new LinkedHashMap<>();
+		for (IResource resource : resources) {
+			if (resource instanceof IBuildConfig) {
```

Comment:
You can actually ask the kind of a resource via resource#getKind()  Not sure it really matters here though